### PR TITLE
Added argument  use_sim_time

### PR DESCRIPTION
**Pull request by ahcorde**
Added argument `use_sim_time`

### CODE REVIEW

File: launch/description.launch.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ def generate_launch_description():
                                         description='The package where the robot description is located'))
     ld.add_action(DeclareLaunchArgument('urdf_package_path',
                                         description='The path to the robot description relative to the package root'))
+    ld.add_action(DeclareLaunchArgument('use_tim_time',
```
Comment (DLu) [inline launch/description.launch.py]: https://getyarn.io/yarn-clip/b40ae11d-b49d-41b0-af4b-765020d77e25

**Review by DLu — COMMENTED** [review]
I think the cleaner way to handle `use_sim_time` post-Foxy is [`SetParameter`](https://answers.ros.org/question/368802/ros2-set-parameters-from-file-and-directly/). Does that work for your use case?

File: launch/description.launch.py (hunk 1 above)
Comment (DLu) [reply launch/description.launch.py]: Just to be explicit, you have written `use_tim_time` in a couple places where you likely meant `use_sim_time` with an s.

**Comment by DLu** [comment]
So there's two ways of utilizing this launch file. If we are launching it from the command line, I don't know what the benefit would be for setting `use_sim_time`. 

If we're including it from another launch file, rather than add the launch argument explicitly, I'm suggesting that instead you could use `SetParameter` before including the launch file and that would set the parameter in all the included nodes without the added complexity in this launch file.

**Comment by ahcorde** [comment]
Now I get the joke, :) 

Feel free to close it if this adds some extra complexity I was not aware of `SetParameter`.

**Pull request closed without merging**